docs/en/guides/apply_catboost_model.md

```diff
@@ -0,0 +1,213 @@
+
+1. [Create a Table](#create-table).
+2. [Insert the Data to the Table](#insert-data-to-table).
+3. [Configure the Model](#configure-model).
+4. [Run the Model Inference from SQL](#run-model-inference).
+
+For more information about training CatBoost models, see [Training and applying models](https://catboost.ai/docs/features/training.html#training).
+
+## Prerequisites {#prerequisites}
+
+If you don't have the [Docker](https://docs.docker.com/install/) yet, install it.
+
+!!! note "Note"
+    [Docker](https://www.docker.com) is a software platform that allows you to create containers that isolate a CatBoost and ClickHouse installation from the rest of the system.
+
+Before applying a CatBoost model:
+
+**1.** Pull the [Docker image](https://hub.docker.com/r/yandex/tutorial-catboost-clickhouse) from the registry:
+
+```bash
+$ docker pull yandex/tutorial-catboost-clickhouse
+```
+
+This Docker image contains everything you need to run CatBoost and ClickHouse: code, runtime, libraries, environment variables, and configuration files.
+
+**2.** Make sure the Docker image has been successfully pulled:
+
+```bash
+$ docker image ls
+REPOSITORY                            TAG                 IMAGE ID            CREATED             SIZE
+yandex/tutorial-catboost-clickhouse   latest              622e4d17945b        22 hours ago        1.37GB
+```
+
+**3.** Start a Docker container based on this image:
+
+```bash
+$ docker run -it -p 8888:8888 yandex/tutorial-catboost-clickhouse
+```
+
+## 1. Create a Table {#create-table}
+
+To create a ClickHouse table for the train sample:
+
+**1.** Start a ClickHouse client:
+
+```bash
+$ clickhouse client
+```
+
+!!! note "Note"
+    The ClickHouse server is already running inside the Docker container.
+
+**2.** Create the table using the command:
+
+```sql
+:) CREATE TABLE amazon_train
+(
+    date Date MATERIALIZED today(), 
+    ACTION UInt8, 
+    RESOURCE UInt32, 
+    MGR_ID UInt32, 
+    ROLE_ROLLUP_1 UInt32, 
+    ROLE_ROLLUP_2 UInt32, 
+    ROLE_DEPTNAME UInt32, 
+    ROLE_TITLE UInt32, 
+    ROLE_FAMILY_DESC UInt32, 
+    ROLE_FAMILY UInt32, 
+    ROLE_CODE UInt32
+)
+ENGINE = MergeTree()
+```
+
+## 2. Insert the Data to the Table {#insert-data-to-table}
+
+To insert the data:
+
+**1.** Exit from ClickHouse console client:
+
+```sql
+:) exit
+```
+
+**2.** Upload the data:
+
+```bash
+$ clickhouse client --host 127.0.0.1 --query 'INSERT INTO amazon_train FORMAT CSVWithNames' < ~/amazon/train.csv
+```
+
+**3.** Make sure the data has been uploaded:
+
+```sql
+$ clickhouse client
+:) SELECT count() FROM amazon_train
+
+SELECT count()
+FROM amazon_train
+
++-count()-+
+|   65538 |
++---------+
+```
+
+## 3. Configure the Model to Work with the Trained Model {#configure-model}
+
+**Optional step**: the Docker container contains model configuration file and link to it in ClickHouse configuration. See `models/amazon_model.xml` and `../../etc/clickhouse-server/config.d/models_config.xml`.
```

Comment:
not clear what exactly is the "optional step", the whole section (3)?

Reply:
Yes. Added note.